packages/manager/modules/pci/src/projects/project/instances/add/add.routing.js

```diff
@@ -25,6 +25,7 @@ export default /* @ngInject */ ($stateProvider) => {
                   ? deals
                   : null,
               )
+              .catch(() => null)
```

Comment:
No error message ?

Reply:
No this is to prevent instance creation crashing just to display an offer. 
An error message could indeed improve this but currently we have no way to display it and it will require translations